setuptools_rust/setuptools_ext.py

```diff
@@ -26,7 +26,10 @@
 if sys.version_info[:2] >= (3, 11):
     from tomllib import load as toml_load
 else:
-    from tomli import load as toml_load
+    try:
+        from tomli import load as toml_load
+    except ImportError:
+        from setuptools.extern.tomli import load as toml_load
```

Comment:
Interesting; I wasn't aware of this namespace and have just asked upstream at https://github.com/pypa/setuptools/pull/4351#issuecomment-2159316805